articles/machines/resources-estimator.md

```diff
@@ -0,0 +1,149 @@
+---
+# Mandatory fields. See more on aka.ms/skyeye/meta.
+title: Quantum Development Kit Resources Estimator | Microsoft Docs 
+description: Overview of Microsoft's Quantum Development Kit Resources estimator 
+author: anpaz-msft
+ms.author: anpaz@microsoft.com 
+ms.date: 1/22/2019
+ms.topic: article
+uid: microsoft.quantum.machines.resources-estimator
+---
+
+# The ResourcesEstimator Target Machine
+
+As the name implies, the `ResourcesEstimator` estimates the resources 
+required to run a given instance of a Q# operation on a quantum computer.
+It accomplishes this by executing the quantum operation without actually 
+simulating the state of a quantum computer; for this reason, 
+it can estimate resources for Q# operations that use thousands of qubits.
+
+## Usage
+
+The `ResourcesEstimator` is just another type of target machine thus 
+it can be used to run any Q# operation. 
+
+As other target machines, to use it on a C# host program create an instance and pass it
+as the first parameter of the operation's `Run` method:
+
+```csharp
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Quantum.MyProgram
+{
+    class Driver
+    {
+        static void Main(string[] args)
+        {
+            ResourcesEstimator estimator = new ResourcesEstimator();
+            var res = MyQuantumProgram.Run(estimator).Result;
```

Comment:
Need to clarify what is the result. At a glance it seems that the result should contain the metrics, not the estimator itself.

Also, what happens if you run another program on the same instance of the estimator? Does it reset the counts from the first run before accumulating the second run?

Reply:
Removed the `.Result` and used `.Wait()` instead. Don't want to get into the details about what is the `Result` and it's not that useful in this context.